### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgExportGraphAsImage.Designer.vb [hunk 1]
```diff
@@ -41,22 +41,22 @@ Partial Class dlgExportGraphAsImage
...
         'cmdBrowse
         '
-        Me.cmdBrowse.Location = New System.Drawing.Point(299, 212)
+        Me.cmdBrowse.Location = New System.Drawing.Point(310, 194)
         Me.cmdBrowse.Name = "cmdBrowse"
         Me.cmdBrowse.Size = New System.Drawing.Size(75, 23)
-        Me.cmdBrowse.TabIndex = 11
-        Me.cmdBrowse.Text = "Browse"
+        Me.cmdBrowse.TabIndex = 4
+        Me.cmdBrowse.Text = "Browse..."
```
Comment (lilyclements) [inline instat/dlgExportGraphAsImage.Designer.vb:48]: Added ellipsis to browse

File: instat/dlgExportGraphAsImage.vb [hunk 2]
```diff
@@ -32,28 +32,32 @@ Public Class dlgExportGraphAsImage
         autoTranslate(Me)
         TestOkEnabled()
     End Sub
+
     Private Sub InitialiseDialog()
+        ' TODO: Help ID
+        ucrInputFile.SetParameter(New RParameter("filename", 0))
```
Comment (lilyclements) [inline instat/dlgExportGraphAsImage.vb:38]: Added parameter positions

File: instat/dlgExportGraphAsImage.vb [hunk 3]
```diff
@@ -32,28 +32,32 @@ Public Class dlgExportGraphAsImage
...
+
         ucrInputFile.Reset()
-        ucrInputFile.SetName("")
         ucrSelectorGraphAsImage.Reset()
+        ucrInputFile.SetName("")
 
-        clsGgsave = New RFunction
-        clsGgsave.SetRCommand("ggsave")
-        ucrBase.clsRsyntax.SetBaseRFunction(clsGgsave)
+        clsggSave.SetPackageName("ggplot2")
```
Comment (lilyclements) [inline instat/dlgExportGraphAsImage.vb:55]: Added `.SetPackageName`

File: instat/dlgExportGraphAsImage.vb [hunk 4]
```diff
@@ -32,28 +32,32 @@ Public Class dlgExportGraphAsImage
...
+        clsggSave.SetRCommand("ggsave")
+        ucrBase.clsRsyntax.SetBaseRFunction(clsggSave)
     End Sub
 
-    Private Sub SetRcodeForControls(bReset As Boolean)
+    Private Sub SetRCodeForControls(bReset As Boolean)
```
Comment (lilyclements) [inline instat/dlgExportGraphAsImage.vb:60]: Capitilised "C"

File: instat/dlgExportGraphAsImage.vb [hunk 5]
```diff
@@ -67,11 +71,7 @@ Public Class dlgExportGraphAsImage
 
     Private Sub ucrBase_ClickReset(sender As Object, e As EventArgs) Handles ucrBase.ClickReset
         SetDefaults()
-        SetRcodeForControls(True)
-        TestOkEnabled()
-    End Sub
-
-    Private Sub ucrInputFile_ControlContentsChanged() Handles ucrInputFile.ControlContentsChanged, ucrSelectedGraphReceiver.ControlContentsChanged
```
Comment (lilyclements) [inline instat/dlgExportGraphAsImage.vb:74]: Removed repeated sub

**Comment by lilyclements** [comment]
@maxwellfundi this is ready to merge

**Comment by maxwellfundi** [comment]
@lilyclements  merged!

**Pull request merged**